casi todos los challenges

sebasWest555 opened the pull request.

sebasWest555 committed:
casi todos los challenge

Complugg3d commented:
for the CSS animation challenge pls take a look on https://www.w3schools.com/howto/howto_css_animate_buttons.asp

Complugg3d commented:
and for the bonus https://www.w3schools.com/html/html5_audio.asp

sebasWest555 committed:
according with ww3schools the audio its like that but i can´t get it yet

sebasWest555 committed:
Update index.html

tha last commit has a cheat code that chatgpt gave me, but i felt guilty and this time i did it by myself

Complugg3d committed:
fixes for the audio challenge

sebasWest555 committed:
Merge pull request #3 from Complugg3d/fix/audio-fixes

fixes for the audio challenge

Complugg3d committed:
- Add css styles
- add js hint to the animate challenge

Complugg3d committed:
remove comment

Complugg3d committed:
remove comment
